implementations/typescript/ockam/ockam_app/src/routes/service/ServiceCreate.svelte

```diff
@@ -3,15 +3,15 @@
   import { invoke } from "@tauri-apps/api/tauri";
 
   let service = "";
-  let port = "10000";
+  let address = "127.0.0.1:10000";
```

Comment:
If are going to support hostnames, I'd change this to `localhost:10000` to make it more readable.